packages/e2e-tests/package.json

```diff
@@ -38,6 +38,6 @@
     "jest-puppeteer": "^9.0.2"
   },
   "peerDependencies": {
-    "puppeteer": "*"
+    "puppeteer": "^21.11.0"
```

Comment:
This can actually stay `*`. The version in the main `package.json` is what matters.